tox.ini

```diff
@@ -1,43 +0,0 @@
-[tox]
-envlist = {py36}-django{111,22,30}, docs, pep8
-
-[testenv]
-whitelist_externals = make
-commands = coverage run --source {envsitepackagesdir}/security --omit="*migrations/*" testing/manage.py test tests
-commands_post = coverage report
-basepython =
-    py36: python3.6
-deps =
-    django-discover-runner
-    django111: django==1.11
-    django22: django==2.2
-    django30: django==3.0
-    coverage
-    ua_parser==0.7.1
-    mock==2.0.0
-
-[testenv:docs]
```

Comment:
Not sure how it was setup before since I don't see it in travis-ci. I'm guessing the repo webhook is setup on the readthedocs.org side. 

@RouganStriker do you have access to this?

Reply:
I don't have access to the readthedocs site so I have no idea how we published the docs before

Reply:
I can yolo it? As in merge and see if the docs generate. If they don't create a separate ticket to look into that and fix it over the next week.

Would be good to deprecate readthedocs entirely from our public repo's and use gitlab pages tbh.

Reply:
https://sdelements.atlassian.net/browse/CI-1073